ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/[externalID]/ResyncModal.tsx

```diff
@@ -14,6 +19,9 @@ type Props = {
 };
 
 export default function ResyncModal({ isOpen, onClose, url }: Props) {
+  const [input, setInput] = useState(url);
+  const [overrideValue, setOverrideValue] = useState(url);
```

Comment:
You shouldn't need to store 2 URL state values -- only the override needs to be stored. The non-overridden value is just the `url` prop.

I think `input` just needs to be a derived value:
```ts
let input = url
if (isURLOverridden) {
  input = overrideValue
}
```